src/components/JobsPage/Jobs.js

```diff
@@ -33,6 +33,7 @@ const Jobs = ({ match, jobsStore, fetchJobs, setSelectedJob, history }) => {
     if (match.params.jobId) {
       let item = jobsStore.jobs.find(item => {
         if (item.uid === match.params.jobId) return item
```

Comment:
There's no reason to return `item` here. The return value of the predicate function provided to `Array.prototype.find` method is considered boolean.

So this is more accurate:
```js
let item = jobsStore.jobs.find(item => {
  if (item.uid === match.params.jobId) return true
  return false
})
```
Which is equivalent to:
```js
let item = jobsStore.jobs.find(item => {
  return item.uid === match.params.jobId ? true : false
})
```
Which is equivalent to:
```js
let item = jobsStore.jobs.find(item => {
  return item.uid === match.params.jobId
})
```
(because the `===` operator simply results in either `true` or `false`)
Which is equivalent to:
```js
let item = jobsStore.jobs.find(item => item.uid === match.params.jobId)
```